packages/app/obojobo-express/server/lti.js

```diff
@@ -717,6 +665,10 @@ const sendHighestAssessmentScore = (
 
 			result.outcomeServiceURL = outcomeData.serviceURL
 
+			// @TODO: move this above getRequiredDataForReplaceResult call
+			// @TODO: heck, why dont we just move all these throws down
+			// to the where the code that finds these issues occurs
+			// and pass them up using the normal throw/catch functionality
 			if (isPreview) {
 				throw Error(ERROR_PREVIEW_MODE)
```

Comment:
Was this supposed to have been moved in this PR, or is it an issue for a future PR?